include/net/cloud.h

```diff
@@ -260,6 +275,44 @@ static inline int cloud_input(const struct cloud_backend *const backend)
 	return backend->api->input(backend);
 }
 
+/**@brief Add Cloud endpoint subscriptions.
+ *
+ * @param backend Pointer to cloud backend structure.
+ * @param endpoint Pointer to a list of endpoint subscriptions.
+ * @param topics_count Number of entries in the endpoint subscription list.
+ */
+static inline int cloud_ep_subscriptions_add(
+			const struct cloud_backend *const backend,
+			struct cloud_endpoint *list, size_t list_count)
+{
+	if (backend == NULL
+	    || backend->api == NULL
+	    || backend->api->ep_subscriptions_add == NULL) {
+		return -ENOTSUP;
+	}
+
+	return backend->api->ep_subscriptions_add(backend, list, list_count);
+}
+
+/**@brief Remove Cloud endpoint subscriptions.
+ *
+ * @param backend Pointer to cloud backend structure.
+ * @param endpoint Pointer to a list of endpoint subscriptions.
```

Comment:
The description does not match the actual argument list.

Reply:
Fixed.